src/tribler/core/components/metadata_store/db/tests/test_torrent_metadata.py

```diff
@@ -393,3 +392,23 @@ def test_popular_torrens_with_metadata_type(metadata_store):
 
     with pytest.raises(TypeError):
         metadata_store.get_entries(popular=True, metadata_type=[REGULAR_TORRENT, CHANNEL_TORRENT])
+
+
+WRONG_TRACKERS_OBJECTS = [
+    (["b'udp://tracker/announce'"]),
+    (None)
```

Comment:
**nitpick**: I think it is better to remove excess parentheses to avoid confusing values with one-element tuples
```suggestion
    ["b'udp://tracker/announce'"],
    None,
```

Reply:
Removed